pkg/clioptions/clusterdiscovery/provider.go

```diff
@@ -67,6 +68,16 @@ func InitializeTestFramework(context *e2e.TestContextType, config *ClusterConfig
 
 	// IPFamily constants are taken from kube e2e and used by tests
 	context.IPFamily = config.IPFamily
+
+	imageStreamString, _, err := exutil.NewCLIWithoutNamespace("").AsAdmin().Run("adm", "release", "info", `-ojsonpath={.references}`).Outputs()
```

Comment:
@eggfoobar this line seems to be failing in the V6 metal jobs breaking them: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/28307/pull-ci-openshift-origin-master-e2e-metal-ipi-ovn-ipv6/1712670143764500480

Reply:
The registry is a local mirror. I cannot find its logs anywhere (sosreport helpfully does not include them), but looking at the preparation step https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/28307/pull-ci-openshift-origin-master-e2e-metal-ipi-ovn-ipv6/1712670143764500480/artifacts/e2e-metal-ipi-ovn-ipv6/baremetalds-devscripts-setup/build-log.txt, I don't see the SHA256 that fails anywhere.

Reply:
https://github.com/openshift/release/pull/44394/files hopefully will do it